lib/spring/test/acceptance_test.rb

```diff
@@ -104,15 +104,14 @@ def without_gem(name)
         end
       end
 
-      test "code changes in pre-referenced app files are picked up" do
```

Comment:
was the old test bad ? <-> leave it and add a new one ? (smaller tests are usually easier to fix/maintain)